src/mapping/parser/grammar.pest

```diff
@@ -0,0 +1,94 @@
+mapping = _{ SOI ~ statement ~ (NEWLINE+ ~ statement)* ~ NEWLINE* ~ EOI }
+
+statement = _{ assignment | function | if_statement }
+
+assignment = { target_path ~ "=" ~ query_arithmetic }
+
+if_statement = {
+    "if" ~ query_arithmetic ~ "{" ~ NEWLINE* ~
+        statement ~
+    NEWLINE* ~ "}" ~ ( "else" ~ "{" ~ NEWLINE* ~
+        statement ~
+    NEWLINE* ~ "}" )?
+}
+
```

Comment:
Would it be worth putting `NEWLINE*` on either side of the brackets so people can indent Allman style? Someone is inevitably going to try.